src/screens/SendwyreInput/SendwyreInput.js

```diff
@@ -0,0 +1,330 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import { ScrollView, Keyboard } from 'react-native';
+import type { NavigationScreenProp } from 'react-navigation';
+import styled, { withTheme } from 'styled-components/native';
+import { connect } from 'react-redux';
+import { SDK_PROVIDER } from 'react-native-dotenv';
+import t from 'tcomb-form-native';
```

Comment:
`tcomb-form-native` hasn't been maintained for over 2 years now. While we use it here and there, I think we should avoid introducing it in new components. The way `formOptions` is structured is confusing, and I also don't like the idea of keeping all of the form data (including constant values) in `this.state`. Do you think you could re-write this to avoid using the lib? You could just import `TextInput` and `SelectorOptions` components directly, and keep just the several changing values in state instead of the whole `formOptions` thing.
But if you feel this is too much work, you can just leave it as it is.